acceptance-tests/README.md

```diff
@@ -2,17 +2,37 @@
 
 This project tests `hs.*` commands as if they were being used by an end-user.
 
+
+
 ## Getting Started
 
 The main test execution is kicked off by running `yarn test-cli` from the root of `hubspot-cli`. This will run the `run-tests` scipe which itself is a CLI, so to see the available options, you can `run-tests --help`. This was done so that the tests can be installed against any given version of the CLI and tested against it. In addition, it was done to keep the test logic separated from the CLI library.
 
+
+
 Note that if you are testing against a QA portal, not a PROD one, you'll need to add the `--qa` flag when running the script. There is still an outstanding issue with this because we attempt to add the `--qa` flag to all `hs` commands, however it is not available for all commands.
 
+
+### Setup
+
+To setup these tests, first [create an app](https://developers.hubspot.com/docs/api/creating-an-app) in a [HubSpot developer account](https://app.hubspot.com/signup-hubspot/crm). Under the "Auth" tab, give your app at least one scope, and set the redirect URL to `localhost:3000/oauth-callback`, and save your changes. Also, take note of your Client ID, and Client Secret and click "Copy full URL".
+![image](https://user-images.githubusercontent.com/48874841/175648758-b2fdb491-90f8-4573-ad9e-af7e1236548c.png)
+
+
+Next, in the navigation bar, click "Testing", and then "Create app test account". Then, paste the OAuth URL you copied earlier into your browser,  choose your newly created test account, and click "Connect App".
+
+Now, navigate back to the "Testing" page, and click your test account. Take note of the Portal ID in the URL (`https://app.hubspot.com/dashboard-library/PORTAL_ID`). Then [navigate to the following page and generate a personal access key](https://app.hubspot.com/portal-recommend/l?slug=personal-access-key) that is associated with your test account.
```

Comment:
Slight nit, we're moving towards "account" over "portal" throughout the repo (there are still references to portal, but we're at least trying to use account in new code). Can this be updated to "Account ID"?